api/src/services/eml-service.ts

```diff
@@ -873,36 +872,6 @@ export class EmlService extends DBService {
     });
   }
 
-  /**
-   * Creates an object representing geographic coverage pertaining to the given project
-   *
-   * @param {IGetProject} projectData
-   * @return {*}  {Record<string, any>}
-   * @memberof EmlService
-   */
-  _getProjectGeographicCoverage(projectData: IGetProject): Record<string, any> {
```

Comment:
Even though we don't use the EML service, if there's a quick + easy way to replace this method with something similar, we should probably do it. Maybe produce a bounding box covering a concatenation of all of the geometries for all of the surveys belonging to this project? Might want to ask Macgreggor

Reply:
Could just replace this call (wherever its used) with a call to `_getSurveyGeographicCoverage`. They return the exact same shape object.

Reply:
Ahh good catch

Reply:
I've updated this part of the code to combine all the associated surveys into a single coverage object